### PR TITLE
Client

### LINKED ISSUE #127: change TotalConnectClient.py to client.py ?
We made a ton of changes to the file structure in the last few PRs, putting each class in it's own file like TotalConnectZone in zone.py.

This is already a major overhaul that anyone using the code (except via HA) will have to review for their implementation.

I'm thinking it's o.k. to also rename TotalConnectClient.py to client.py to be in line with all of the other files.

Thoughts?

### PR DESCRIPTION
**Pull request by austinmroczek**
fix #127 by changing TotalConnectClient.py to client.py to be like all of the other files.

### CODE REVIEW

File: total_connect_client/client.py [hunk 1]
```diff
@@ -209,7 +204,9 @@ def authenticate(self):
         self.locations will not be refreshed if it was non-empty on entry.
         """
         if self._invalid_credentials:
-            LOGGER.error(f"not authenticating: password already failed for user {self.username}")
+            LOGGER.error(
+                f"not authenticating: password already failed for user {self.username}"
```
Comment (dulitz) [inline total_connect_client/client.py:208]: I think this was less than 100 characters before? vertical space is also a valuable resource.

File: total_connect_client/client.py [hunk 2]
```diff
@@ -307,12 +304,16 @@ def _make_locations(self, response):
...
+                location_id
+            ].auto_bypass_low_battery = self.auto_bypass_low_battery
 
             # set the usercode for the location
-            usercode = (self.usercodes.get(location_id) or
-                        self.usercodes.get(str(location_id)) or
-                        self.usercodes.get('default'))
+            usercode = (
+                self.usercodes.get(location_id)
+                or self.usercodes.get(str(location_id))
```
Comment (dulitz) [inline total_connect_client/client.py:314]: I see what you're doing here, but it's a very unusual style for Python, and it consumes two lines of vertical space.

**Comment by dulitz** [comment]
I made a few comments, but this is a **big** improvement and I think you should merge it before the release.

File: total_connect_client/client.py (hunk 2 above)
Comment (austinmroczek) [reply total_connect_client/client.py:314]: I'm just using the default `black` auto-formatting.  I'm just following what's done in the HA codebase.

Comment (dulitz) [reply total_connect_client/client.py:314]: That makes sense then. If you start an unrelated project that's one rule I would strongly suggest tweaking.

Comment (austinmroczek) [reply total_connect_client/client.py:314]: There may be settings for black that we can tweak.

**Pull request merged**